### DISCUSSION TITLE
Search in website

### DISCUSSION THREAD

**Discussion by Rar9** [themesberg/flowbite-svelte#423] [Ideas]
It would help to be able to search for things like Button or other available collections.

**Comment by shinokada** [comment]
[This](https://mango-dune-07a8b7110.1.azurestaticapps.net/?repo=themesberg%2Fflowbite-svelte) may be helpful.

**Comment by Rar9** [comment]
Not really, just something similar to what tailwindcss / daizyui offers in terms of search.

A great enhancement would be to change color/edit on site to make cut /paste faster as well.
